### PR TITLE
BXMSPROD-883 pull request flow on github actions

### PR DESCRIPTION
**Pull request by Ginxo**
https://issues.redhat.com/browse/BXMSPROD-883

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  build-chain:
+    runs-on: ubuntu-latest
+    name: Build Pull Request
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build Chain
+      id: build-chain
+      uses: Ginxo/github-action-build-chain@master
+      with:
+        parent-dependencies: 'optaweb-employee-rostering'
```
Comment (yurloc) [inline .github/workflows/pull_request.yml:11]: Why not `parent-dependencies: 'optaplanner'`? This project doesn't depend on optaweb-employee-rostering.

Comment (mareknovotny) [reply .github/workflows/pull_request.yml:11]: I agree this dependency definition is not correct, this comes only from the [repository-list.txt](https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/script/repository-list.txt). The logical dependency is optaplanner as @yurloc suggested

File: .github/workflows/pull_request.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+jobs:
+  build-chain:
+    runs-on: ubuntu-latest
+    name: Build Pull Request
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build Chain
+      id: build-chain
+      uses: Ginxo/github-action-build-chain@master
```
Comment (mareknovotny) [inline .github/workflows/pull_request.yml:9]: Could we move this to kiegroup organization?

Comment (Ginxo) [reply .github/workflows/pull_request.yml:9]: yes we can, moving it

File: .github/workflows/pull_request.yml (hunk 1 above)
Comment (Ginxo) [reply .github/workflows/pull_request.yml:11]: this is something pending from my side. It also depends on kie-wb-common, for instance. Just trying to reproduce the same behavior we have in Jenkins and once it is stable I will add the proper parent dependencies. Do you agree?

File: .github/workflows/pull_request.yml (hunk 2 above)
Comment (Ginxo) [reply .github/workflows/pull_request.yml]: @mareknovotny done!

File: .github/workflows/pull_request.yml (hunk 1 above)
Comment (yurloc) [reply .github/workflows/pull_request.yml:11]: Yes, that makes sense. Let's first reproduce the original process and then optimize it.

**Pull request merged**